File: Examples/Access/VB/Examples/AccessExamples/FormMain.vb
Public Class FormMain

    Public Sub New()

        InitializeComponent()

        Me.Text = "NetOffice Access Examples in Visual Basic"
        LoadExamples()

    End Sub

    Private Sub LoadExamples()

        LoadExample(New Example01())
        LoadExample(New Example02())
        LoadExample(New Example03())
        LoadExample(New Example04())
        LoadExample(New Example05())

    End Sub

End Class

File: Examples/Access/Automation/FormMain.cs
using System;
using ExampleBase;

namespace AccessExamplesCS4
{
    public partial class FormMain : ExampleForm
    {
        public FormMain()
        {
            InitializeComponent();
            this.Text = "NetOffice Access Examples in C#";
            LoadExamples();
        }

        private void LoadExamples()
        {
            LoadExample(new Example01());
            LoadExample(new Example02());
            LoadExample(new Example03());
            LoadExample(new Example04());
            LoadExample(new Example05());
        }
    }
}